packages/core/src/plugins/router/routes/index.ts

```diff
@@ -72,7 +72,7 @@ function validateAction(actionLike: ServiceAction | { default: ServiceAction },
     throw new ValidationError(`action.auth in ${route} must be a string or an object`)
   }
 
-  if (is.defined(schema) && !is.string(schema)) {
+  if ((schema !== undefined && schema !== null && schema !== false) && !is.string(schema)) {
```

Comment:
```suggestion
  if (schema && typeof schema !== 'string') {
```

в целом тоже самое, бонусом `''` будет игнорироваться тк это  при coerce false